pkg/crc/services/dns/template.go

```diff
@@ -59,13 +64,27 @@ func createDnsmasqDNSConfig(serviceConfig services.ServicePostStartConfig) error
 	if err != nil {
 		return err
 	}
+
+	dnsHostConfig, err := createDnsConfigFile(dnsmasqConfFileValues, dnsmasqHostsTemplate)
+	if err != nil {
+		return err
+	}
+
+	encodeddnsConfig = base64.StdEncoding.EncodeToString([]byte(dnsHostConfig))
+	_, err = serviceConfig.SSHRunner.Run(
+		fmt.Sprintf("echo '%s' | openssl enc -base64 -d | sudo tee /var/srv/hosts.openshift > /dev/null",
+			encodeddnsConfig))
+	if err != nil {
```

Comment:
Instead of duplicating this a 3rd time, can we add a helper?
https://github.com/cfergeau/crc/commit/51a0ad953873d2f8375a52a4addfdba203ab17f5